buildenv/jenkins/README.md

```diff
@@ -68,14 +68,16 @@ This folder contains Jenkins pipeline scripts that are used in the OpenJ9 Jenkin
         - ppc64le_linux, s390x_linux, x86-64_linux, x86-64_linux_xl, ppc64_aix, x86-64_windows, x86-32_windows, x86-64_mac
 
 - OpenJ9 committers can request builds by commenting in a pull request
-    - Format: `Jenkins <build type> <level>.<group> <platform>[,<platform>,...,<platform>] jdk<version>[,jdk<version>,...,jdk<version>]`
+    - Format: `Jenkins <build type> <level>.<group>+<test_flag> <platform>[,<platform>,...,<platform>] jdk<version>[,jdk<version>,...,jdk<version>]`
```

Comment:
You could wrap it in [ ] to indicate its optionality, even as you also state it is optional (much like you have done with comma-separated extra platforms.

Reply:
updated